autoscale/src/main/java/com/sequenceiq/periscope/service/EntitlementValidationService.java

```diff
@@ -31,4 +31,20 @@ public boolean autoscalingEntitlementEnabled(String accountId, String cloudPlatf
         }
         return entitled;
     }
+
+    @Cacheable(cacheNames = "accountEntitlementCache", key = "{#accountId,#cloudPlatform}")
+    public boolean stopStartAutoscalingEntitlementEnabled(String accountId, String cloudPlatform) {
+        boolean entitled = autoscalingEntitlementEnabled(accountId, cloudPlatform);
+        if (!entitled) {
+            return false;
+        }
+        if ("AWS".equalsIgnoreCase(cloudPlatform)) {
```

Comment:
We already have these constants defined in the `com.sequenceiq.cloudbreak.common.mappable.CloudPlatform` enum. Can you please use them?

Reply:
Will change this.

Reply:
Fixed in the next push. Resolving conversation.